Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/cache/downloader/Logging.kt

```diff
@@ -0,0 +1,15 @@
+package com.github.adamantcheese.chan.core.cache.downloader
```

Comment:
You can toss these functions into the Logger class directly, no need for another logging class.

Reply:
Nah, I wanted it this way specifically so it prints the current thread it is executed on. They are internal for that package so you won't be able to use them in any other places of the project.